res/lang/news/news_en.txt

```diff
@@ -1,11 +1,11 @@
-NEWS_TECHNICS_MEDIA   = Media / Technics
+NEWS_TECHNICS_MEDIA   = Media / Technology
 NEWS_POLITICS_ECONOMY = Politics / Economy
 NEWS_SHOWBIZ          = Showbiz
-NEWS_SPORT            = Sport
+NEWS_SPORT            = Sports
 NEWS_CURRENTAFFAIRS   = Current affairs
 NEWS_CULTURE          = Culture
 
-NEWS_SORT_TOPICALITY = Sort by topicality
+NEWS_SORT_TOPICALITY = Sort by freshness
```

Comment:
to discuss with @GWRon : topicality was chosen intentionally

Reply:
That was one of the most irritating things I came across, so much so that I put it in the opening text of this entire PR.

> For example, "Topicality" is not a very intuitively understandable word. "Recency" is often used in such context (though also may be confusing in this special context), but why not "Freshness". ("Fresh" or just "Hot" might work too.)

"Topicality" means "aktuell" more in the sense of something being on-topic, "auf der aktuellen Tagesordnung".

Reply:
this is a quote from a discord discussion I had some days ago:

myself:
> I have a short question about an English term. 
>
>In my game I use the term "topicality" to describe how "new" people perceive something. A news with high "topicality" just happened and was not broadcasted that often. So not many people know it already. 
> A movie which has been seen by many people has a lower "topicality" and so on.
> 
> A user now proposed to rename it to "freshness", yet I have concerns that it sounds less "serious".
>
>Can a mother tongue help me here? What terms exist to describe that "perceived new"?

Reply by an Englishman (so native English speaker):
> Topicality is a pretty good word for the purpose and you can use Topic or Topical to describe it. Another term that might work is Prevailing/Prevalent, but mostly we see these advertised as "New", "Latest" or "Current". I can see where "Freshness" comes from, but it doesn't really feel right.


myself:
> Prevailing sounds a bit like ... "dominance"
> for movies the term needs to describe: "Fresh from production", "straight out of cinema", "just produced" ... and the value lowers the older the programme becomes (known by "word of mouth", watched on tape/disc/stream). In the game the value lowers for now by "age" (to describe the "watched somewhere") but also for each broadcast one of the channels did. The "age" modifies the "maximum" of that value (wont get any higher). The broadcasts lower this value a tiny bit too - but also lower a "temporary topicality". This value refreshes each day a bit. This is to ressemble the "memory" of the audience. They forget again about the movie ...
> This is why I wrote about a "perceived" feeling of being "new"


Maybe this helps to clarify what _I_ intented with the term "topicality" and maybe assists in finding a better term.
"main concept"-terms (reach, topicality, "movie agency"/room names with a "task" connected to them, ...)  need that kind of discussion, so excuse if we do them here in the PR (think they fit better in the discussion-area or within a new issue).

Reply:
moving discussion to https://github.com/TVTower/TVTower/issues/892

Reply:
As the changes were rolled back, I resolve this conversation.